src/scripts.js

```diff
@@ -97,12 +94,16 @@ function handleMetricSubmits(event) {
       body: JSON.stringify(dataToPost),
     })
     .then(response => response.json())
-    .then(message => handleHydrationSuccess(message))
+    .then(message => handlePostSuccess('.hydration'))
     .catch(error => console.log(error.message))
   }
 
-  function handleHydrationSuccess(message) {
+  function handlePostSuccess(className) {
+    let form = document.querySelector(`${className}`)
     loadMonitorData()
+    displaySuccessMessage(form)
+    form.reset();
+    setTimeout(function() { form.lastChild.innerHTML = '' }, 5000);
```

Comment:
Glad we made this dynamic, may want to handle other functions in a similar manner. This works beautifully too.  Good job here with the timeout.